.project-keeper.yml

```diff
@@ -5,6 +5,34 @@ sources:
       - maven_central
       - integration_tests
 excludes:
-  # Terraform setup
+  # Environment "aws". Will be fixed in https://github.com/exasol/project-keeper/issues/566
   - "E-PK-CORE-18: Outdated content: '.github/workflows/ci-build.yml'"
-linkReplacements:
+build:
```

Comment:
Maybe add a comment: 
This will take effect only after upgrading PK to version x.y.z?